### DISCUSSION TITLE
Golden Cheetah

### DISCUSSION THREAD

**Discussion by posssson** [doudar/SmartSpin2k#467] [Q&A]
Hi everyone, I try to use SmartSpin with Golden Cheetah but impossible to see SS2K in the Cheetah interace. Someone succeded it ?

**Comment by eMadman** [comment]
I tried Golden Cheetah in the past with my old IC4 and power meter as well with no real luck.  This was over a year and a half ago.  Because you can't do explicit pairing in Golden Cheetah, it's really hard to know if it sees the device at all.

I think you may have more luck testing it in Linux but I don't have a dedicated Linux box to test with.

Separately, they have this in their issue tracker: https://github.com/GoldenCheetah/GoldenCheetah/issues/4318

**Reply by doudar** [reply]
I've unfortunately never been able to get Golden Cheetah to work with any power meter either. My data is a couple years old, both Linux and windows.

**Comment by posssson** [comment]
Thanks for the quick answer. Currently I use MyWoosh but I want to change cause of lot of bug and also becasue after each update of the app I need to download all the map again so do you know an app it can be without graphic video to controle smartspin with workouts ?

**Reply by doudar** [reply]
I tried Flux once and it works really good! It's an awesome project:

https://github.com/dvmarinoff/Flux

**Reply by posssson** [reply]
Yes I spoke about Flux in the SmartSpin discusion some month ago but it is very basic and the connection is not very stable.

**Reply by doudar** [reply]
Maybe this? I haven't messed around with it.

https://github.com/davidzof/wattzap-ce

**Reply by posssson** [reply]
Ok I just found https://github.com/OlliV/bfree I will try this.

**Reply by eMadman** [reply]
@posssson, take a look at this as well.  It's been quite useful for testing on my side: https://github.com/zoffline/zwift-offline

**Reply by eMadman** [reply]
@posssson - there's one more if you have Grand Theft Auto V (a bit buggy but quite a bit of fun!) https://www.gtbikev.com/

**Reply by posssson** [reply]
Waouhh Incredible!! Some people live 100 years in the future.
